dhis-2/dhis-api/src/main/java/org/hisp/dhis/calendar/impl/NepaliCalendar.java

```diff
@@ -144,25 +145,36 @@ public int isoWeek(DateTimeUnit dateTimeUnit) {
 
   @Override
   public int week(DateTimeUnit dateTimeUnit) {
-    return isoWeek(dateTimeUnit);
+    /*
+     * https://en.m.wikipedia.org/wiki/ISO_week_date
+     */
+    int week = (10 + getDayOfYear(dateTimeUnit) - isoWeekday(dateTimeUnit)) / 7;
```

Comment:
Could you add constants to the integers here?
Also, a Javadoc would be nice. The Wikipedia website can be left as a reference.